student.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name: 'Unknown', parent_permission: true, classroom: 'Unknown')
+    super(age, name: name, parent_permission: parent_permission)
```

Comment:
- Optional
You can directly pass the parameters without setting them again if you are passing them with the same name. Here you do like `super(age, name, parent_permission)`. It would be the same as you did above.